schema/deploy/shipping/views.sql

```diff
@@ -4177,7 +4177,8 @@ create or replace view shipping.linelist_data_for_wa_doh_v1 as (
               when present = 'f' then 'negative'
               else 'inconclusive'
           end as test_result,
-          hcov19_result_release_date as date_tested,
+          result_ts as date_tested,
+          hcov19_result_release_date,
```

Comment:
We can discuss over in https://github.com/seattleflu/backoffice/pull/312, but I wonder if we'd benefit from making a new _v2 of the shipping view.